services/speech_to_text/v1.js

```diff
@@ -288,22 +296,30 @@ module.exports = function (RED) {
       } else if (urlCheck(msg.payload)) {
         temp.open({suffix: '.audio'}, function(err, info){
           if (err) {
-            this.status({fill:'red', shape:'ring', 
+            node.status({fill:'red', shape:'ring', 
               text:'unable to open url audio stream'});          
             message = 'Node has been unable to open the url audio stream'; 
 
             node.error(message, msg);
             return;        
           }  
 
-          stream_url(info.path, msg.payload, function (format) {
+          stream_url(info.path, msg.payload, function (err, format) {
+            if (err) {
+              node.status({fill:'red', shape:'ring', 
+                text:'url stream not recognised as audio'});          
+                message = 'Node did not recognise the url audio stream as audio'; 
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 14 space characters but found 16.](https://www.codacy.com/app/BetaWorks-NodeRED-Watson/node-red-node-watson/file/2697377886/issues/source?bid=3330694&fileBranchId=3341077#l311)